k8s/postgresql/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,141 @@
 actions:
-- name: dummy test
+- name: secure login
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-postgresql-svc"
+      IP=''
+      PGPASSWORD=''
+      FAILED=0
```

Comment:
Please do not concatenate together test cases - if there is counter 'FAILED', it means that one bashTest involves multiple test cases. I see that in this test case there is only one test after waiter, but why number it as #1 then?

Reply:
Just a name. Would you like to change it to Test instead of Test1?